datahub-web-react/src/app/entity/shared/constants.ts

```diff
@@ -7,6 +7,7 @@ export const REDESIGN_COLORS = {
 export const ANTD_GRAY = {
     1: '#FFFFFF',
     2: '#FAFAFA',
+    2.5: '#F8F8F8',
```

Comment:
is a new grey really necessary? We've made it this far with the grays we have

Reply:
Synced offline! Will keep it here.